h2o-automl/src/main/java/ai/h2o/automl/targetencoding/TargetEncoder.java

```diff
@@ -508,6 +541,18 @@ public void map(Chunk cs[]) {
       }
     }
 
+  public Frame applyTargetEncoding(Frame data,
```

Comment:
We needed target encoding to respect key of the frame created when necessary. This method is here to represent the old API and not break anything. Used mainly by tests and maybe by Sparkling water.